src/Rings/groebner.jl

```diff
@@ -151,57 +151,68 @@ with respect to the ordering
 ```
 """
 function standard_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_ordering(base_ring(I)),
-                        complete_reduction::Bool = false, algorithm::Symbol = :buchberger) 
+                        complete_reduction::Bool = false, algorithm::Symbol = :default)
```

Comment:
This change should be reflected in the docstring as well

Reply:
@ederc can you please address this comment? Otherwise this PR seems fine.

Reply:
Updated.